images-list

```diff
@@ -3,8 +3,6 @@
 # https://github.com/rancher/image-mirror/pull/27
 #
 # LIST FORMAT: <SOURCE> <DESTINATION> <TAG>
-coredns/coredns rancher/coredns-coredns 1.7.0
```

Comment:
We already pushed 1.7.0 so it should stay on the list just so that we know what we've done. 1.7.1 never got pushed since the source doesn't exist, so I'm OK with removing that one.

Reply:
Updated to only remove 1.7.1 - makes sense to keep a record!